test/dynamicHMC.jl

```diff
@@ -16,7 +16,9 @@ t = collect(range(1,stop=10,length=10))   # observation times
 sol = solve(prob1,Tsit5())
 randomized = VectorOfArray([(sol(t[i]) + σ * randn(2)) for i in 1:length(t)])
 data = convert(Array,randomized)
-transform = (a = asℝ₊,)
+function transform(p)
```

Comment:
I guess I was not clear (sorry): I would just design the API so that the user can provide the transformation directly, eg as
```julia
bayesian_result = dynamichmc_inference(prob1, Tsit5(), t, data, [Normal(1.5, 1)], as((a = asℝ₊,))
```
as transformations are already callable.

Apologies for the documentation of the related packages being so WIP, PRs are welcome.